### DISCUSSION TITLE
linux newb here... how do I save prompt in plandex tell?

### DISCUSSION THREAD

**Discussion by MokshaK** [plandex-ai/plandex#115] [Q&A]
I tried usual shortcuts like control o, control z, control s, etc, w/o luck.

thank you

**Comment by MokshaK** [comment]
nvm I learned that default editor was Vim and googled my way to solution. thank you

**Reply by danenania** [reply]
I'm glad you found a solution. Another option if you want to use a different editor is to create a text file with your prompt in it then pass it in with the `--file/-f` flag.

```bash
plandex tell -f prompt.txt
```

You can also pass prompts inline. This is best for short prompts since they can be difficult to edit.

```bash
plandex tell 'your prompt'
```
